packages/notebook/src/widget.ts

```diff
@@ -647,6 +647,14 @@ export class StaticNotebook extends Widget {
     const cell = this.contentFactory.createMarkdownCell(options, this);
     cell.syncCollapse = true;
     cell.syncEditable = true;
+    // Connect collapsed signal for each markdown cell widget
+    if(this instanceof Notebook) {
```

Comment:
Could you elaborate on the need for that test? As the method is `private` it should not be required.

Reply:
Hm, seems this is a remnant of me trying to cast in type script. Sorry, new to this language. I'll fix that.